packages/MarkdownEditor-Tests.package/MarkdownInlineParserTest.class/instance/testInterpretTokensBold.st

```diff
@@ -0,0 +1,16 @@
+tests - interpretation
+testInterpretTokensBold
+
+	| tokens |
+	tokens := OrderedCollection new.
+	tokens add: (MarkdownDelimiterEmphasis new content: '**'; position: 1; leftFlanking: true; rightFlanking: false; yourself).
+	tokens add: (MarkdownString new content: 'f'; position: 3; yourself).
```

Comment:
Why do we set a `position` on the string/char tokens? We don't need them like shown in `testDelimiterCantCloseItself`.